src/css/components/_button.scss

```diff
@@ -13,7 +13,6 @@
 
 .button--blue {
   background: var(--blue);
-  margin-bottom: 10px;
```

Comment:
Wasn’t this added by something that was already merged?

Reply:
Sorry, I didn't get you! But this line of code isn't present in my latest pull(sync fork) too, meanwhile, I added it again and removed mentioned unnecessary spaces, you may have a look

Reply:
It was added in this pull of yours: https://github.com/zaproxy/zaproxy-website/pull/1758/files why would we remove it in this one?

Reply:
it wasn't supposed to be removed I think it was a mistake, So I just restored it & a couple of other fixes too.
sorry for petty mistakes 😅, 
gaps are now pretty much adjusted. Still, if there's any left, let me know!

Reply:
You haven't pushed your changes.

Reply:
> You haven't pushed your changes.

done